cli/state/help.py

```diff
@@ -2,3 +2,7 @@
 Provide help messages for command line interface's state commands.
 """
 STATE_ACCOUNT_ADDRESS_ARGUMENT_HELP_MESSAGE = 'Account address to get a state by.'
+STATE_START_ADDRESS_ARGUMENT_HELP_MESSAGE = 'Account address to get a list of states starting from.'
```

Comment:
Make constant `STATES_ACCOUNT_ADDRESS_ARGUMENT_HELP_MESSAGE` and replace `STATE` with `STATES` in other constants.

Reply:
@anastasiia-bilova, done.